src/client/app/components/BaseLayout/BaseLayoutHeader.tsx

```diff
@@ -176,6 +192,14 @@ const BaseLayoutHeader: FunctionComponent<BaseLayoutHeaderProps> = ({
       public: true,
       icon: githubIcon,
     },
+    {
+      text: 'Verify Messages',
+      link: i18next.t('general.links.verifyMessages'),
+      public: true,
+      icon: githubIcon,
+      mobileOrder: 6,
```

Comment:
should we remove `mobileOrder: 6`? `mobileOrder` only seems applicable for non-public pages (after login)

Reply:
edit: fixed in https://github.com/opengovsg/GoGovSG/pull/2199/commits/6842e7ad2da1c1430c3403e22d803de6c1d6c826